src/examples/src/Example.m.css

```diff
@@ -3,3 +3,9 @@
 	overflow-x: hidden;
 	overflow-y: scroll;
 }
+
+.iframe {
+	width: 100%;
+	height: 95%;
```

Comment:
why 95%?

Reply:
any bigger and it results in scrollbars within scrollbars